### PR TITLE
iOS : Fix SONYRT-1667 - Remove Padding from the top of the view

### PR DESCRIPTION
**Pull request by septronic2**
### Platforms affected

iOS

### What does this PR do?

Removes the toolbar entirely, and replaces the defined toolbar height and footer height to properties of the CDVBrowserOptions, so that they can be changed according to project requirements

### What testing has been done on this change?

tested on both iPhone and iPad

### Checklist
- [ ] [ICLA](http://www.apache.org/licenses/icla.txt) has been signed and submitted to secretary@apache.org.
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by septronic2** [comment]
@massivekarenwong Hi Karen, This is the PR for removing the toolbar ghost padding from the top of the inappbrowser.

File: src/ios/CDVInAppBrowser.m [hunk 1]
```diff
@@ -28,9 +28,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 #define    kInAppBrowserToolbarBarPositionBottom @"bottom"
 #define    kInAppBrowserToolbarBarPositionTop @"top"
 
-#define    TOOLBAR_HEIGHT 44.0
 #define    LOCATIONBAR_HEIGHT 21.0
-#define    FOOTER_HEIGHT ((TOOLBAR_HEIGHT) + (LOCATIONBAR_HEIGHT))
```
Comment (massivekarenwong) [inline src/ios/CDVInAppBrowser.m:33]: Keep these constants and in setDefaultValues use these constants

File: src/ios/CDVInAppBrowser.m [hunk 2]
```diff
@@ -1241,6 +1241,11 @@ -(void)setDefaultValues
 {
 	NSNumberFormatter* numberFormatter = [[NSNumberFormatter alloc] init];
 	[numberFormatter setAllowsFloats:YES];
+	[self setValue:[NSNumber numberWithBool:NO] forKey:@"statusbarstylelight"];
+	[self setValue:[NSNumber numberWithInteger:44] forKey:@"toolbarheight"];
```
Comment (massivekarenwong) [inline src/ios/CDVInAppBrowser.m]: update this line to use TOOLBAR_HEIGHT

File: src/ios/CDVInAppBrowser.m [hunk 3]
```diff
@@ -1241,6 +1241,11 @@ -(void)setDefaultValues
...
+	NSNumber *footNumber = [self valueForKey:@"toolbarheight"];
+	NSInteger footerCalculated = [footNumber integerValue] + LOCATIONBAR_HEIGHT;
+	[self setValue:[NSNumber numberWithInteger:footerCalculated] forKey:@"toolbarFooterHeight"];
```
Comment (massivekarenwong) [inline src/ios/CDVInAppBrowser.m]: update this line to use FOOTER_HEIGHT

**Review by massivekarenwong — CHANGES_REQUESTED** [review]

File: src/ios/CDVInAppBrowser.m [hunk 4]
```diff
@@ -1241,6 +1243,10 @@ -(void)setDefaultValues
 {
 	NSNumberFormatter* numberFormatter = [[NSNumberFormatter alloc] init];
 	[numberFormatter setAllowsFloats:YES];
+	[self setValue:[NSNumber numberWithBool:NO] forKey:@"statusbarstylelight"];
+	[self setValue:[NSNumber numberWithInteger:TOOLBAR_HEIGHT] forKey:@"toolbarheight"];
+	NSInteger footerCalculated = TOOLBAR_HEIGHT + LOCATIONBAR_HEIGHT;
+	[self setValue:[NSNumber numberWithInteger:footerCalculated] forKey:@"toolbarFooterHeight"];
```
Comment (massivekarenwong) [inline src/ios/CDVInAppBrowser.m]: You don't need this footerCalculated integer, just use FOOTER_HEIGHT here

**Review by massivekarenwong — CHANGES_REQUESTED** [review]

File: src/ios/CDVInAppBrowser.h [hunk 5]
```diff
@@ -76,6 +76,8 @@
 @property (nonatomic, copy) NSString* alphagradient2;
 @property (nonatomic, copy) NSString* toolbarcolor;
 @property (nonatomic, assign) BOOL hideallbuttons;
+@property (nonatomic) CGFloat toolbarheight;
+@property (nonatomic) CGFloat toolbarFooterHeight;
```
Comment (massivekarenwong) [inline src/ios/CDVInAppBrowser.h]: Can you add some description in the README about the newly added toolbarheight and toolbarfooterheight option? Remember to clarify that it is only available on iOS. Also, change the camel case "toolbarFooterHeight" to "toolbarfooterheight" to keep it consistent with the other options

**Review by massivekarenwong — CHANGES_REQUESTED** [review]

**Review by massivekarenwong — APPROVED** [review]

**Pull request merged**